cpp/include/cudf/types.h

```diff
@@ -165,6 +165,10 @@ typedef struct gdf_context_{
   int flag_distinct;            /**< for COUNT: DISTINCT = 1, else = 0 */
   int flag_sort_result;         /**< When method is GDF_HASH, 0 = result is not sorted, 1 = result is sorted */
   int flag_sort_inplace;        /**< 0 = No sort in place allowed, 1 = else */
+  int flag_groupby_include_nulls; /**< 0 = Nulls are ignored in group by keys (Pandas style), 
+                                      1 = Nulls are treated as values in group by keys where NULL == NULL (SQL style)*/ 
+  int flag_nulls_sort_behavior; /**< 0 = Nulls are are treated as largest, 1 = Nulls are treated as smallest, 
```

Comment:
Multi-valued options, which are not meaningful as actual numbers, should use enum classes. If they're supposed to be C-compatible - then plain enums.

Also, same comment regarding the magic numbers. Yes, I know other fields in `gdf_cotext` may already have this affliction...

Reply:
Agreed, a `typedef enum` would be appropriate here.